spock-tests/pom.xml

```diff
@@ -0,0 +1,87 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <artifactId>blueflood</artifactId>
+    <groupId>com.rackspacecloud</groupId>
+    <version>2.1.1-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>spock-tests</artifactId>
+  <name>Blueflood Spock Tests</name>
+  <url>http://blueflood.io</url>
+
+  <dependencies>
+    <!-- Spock Framework -->
```

Comment:
Wouldn't it be better to add groovy and spock to the entire project? That way we can write spock tests anywhere in the future.

Reply:
DOne